plenoptic/synthesize/eigendistortion.py

```diff
@@ -428,14 +429,18 @@ def plot_distorted_image(self,
         Parameters
         ----------
         eigen_index: int
-
+            Index of eigendistortion to plot. E.g. If there are 10 eigenvectors, 0 will index the first one, and
+            -1 or 9 will index the last one.
         alpha: float, optional
             Amount by which to scale eigendistortion for image + (alpha * eigendistortion) for display.
         process_image: Callable
             A function to process the image+alpha*distortion before clamping between 0,1. E.g. multiplying by the
             stdev ImageNet then adding the mean of ImageNet to undo image preprocessing.
         ax: matplotlib.pyplot.axis, optional
             Axis handle on which to plot.
+        plot_complex: str, optional
+            Parameter for ``plenoptic.imshow`` determining how to handle complex values. Defaults to 'rectangular', i.e.
+            not complex. Can also be 'polar' or 'logpolar'; see method for details.
```

Comment:
rectangular isn't "not complex", it plots the real and imaginary as separate components. 

Also, instead of "see method for details", I think we should say "see that method's docstring" or something similar.

Reply:
That last point is really minor -- how is it done in other libraries?

Reply:
done. i agree w/ ur second point since it's more explicit.